compiler/noirc_frontend/src/tests/visibility.rs

```diff
@@ -28,29 +28,38 @@ fn errors_once_on_unused_import_that_is_not_accessible() {
         ))
     ));
 }
+
+fn assert_type_visibility_error(src: &str, private_typ: &str, public_item: &str) {
```

Comment:
Should this be named something like `assert_type_is_more_private_than_item_error`?

Reply:
Okay, renamed. I'm notoriously bad at naming 🙏